tests/test_basic.py

```diff
@@ -2,16 +2,23 @@
 
 import unittest
 import warnings
+import os
+import sys
 
 import ebi.ols.api.helpers as helpers
 from ebi.ols.api.base import ListClientMixin
 from ebi.ols.api.client import OlsClient
 
+# warnings.simplefilter("ignore", ResourceWarning)
```

Comment:
Changes in this file dosn't seem to be connected to the code coverage changes which are the main topic of this PR. The edits here look fine (I think), but just wanted to check that they should be part of the commit.

Reply:
Weel this change should not be part of the commit, just another thing I tried in the meantime, to get rid of (very) annoying warnings when using standard python unittests instead of nosetests. And of which I forgot to remove the modification. Thanks, @james-monkeyshines  for  pointing it